src/services/tripsService.js

```diff
@@ -83,6 +86,29 @@ class tripHelper {
 
     return allTrips;
   }
+
+  /**
+   * @param {string} page
+   * @param {string} limit
+   * @returns {object} most traveled centres infos
+   */
+  static async getTrendingCentres(page, limit) {
+    const pagination = paginate(page, limit),
+      mostTraveledCentres = await Booking.findAll({
+        group: ['accommodationId', 'Accommodation.id'],
+        attributes: ['accommodationId', [Sequelize.fn('COUNT', 'accommodationId'), 'peopleVisited']],
+        include: [{
+          model: Accommodations,
+          attributes: {
+            exclude: ['createdBy', 'id', 'createdAt', 'updatedAt']
+          }
+        }],
+        order: [[Sequelize.col('peopleVisited'), 'DESC']],
```

Comment:
where is this collum coming from?  col `[peopleVisited]`

Reply:
sorry about the approval things 
I thought that I was making a comment

Reply:
@blaise82 ```['peopleVisited']``` is an alias that stands for how many bookings were made on a particular groud of accommodations. So the reason why I am using it in ```ORDER BY`` it's because I wanna filter the accommodations with more visitors like so